openshift/template.yaml

```diff
@@ -96,7 +96,7 @@ parameters:
   displayName: Docker image
   required: true
   name: DOCKER_IMAGE
-  value: "bayesian/3scale-admin-gateway"
+  value: "bayesian/f8a-3scale-admin-gateway"
```

Comment:
Please go with `fabric8-analytics/3scale-admin-gateway` here. Makefile will produce image with that name (no `f8a-` prefix here).